### PR TITLE
Border-top for iOS/embedded case

### PR DESCRIPTION
**Pull request by dvoytenko**

### CODE REVIEW

File: src/service/fixed-layer.js [hunk 1]
```diff
@@ -201,6 +203,8 @@ export class FixedLayer {
           // to be styled with `0`.
           const isTransferrable = (
               isFixed &&
+              visibility != 'hidden' &&
```
Comment (cramforce) [inline src/service/fixed-layer.js]: Please add comment

File: src/service/viewport-impl.js [hunk 2]
```diff
@@ -878,7 +878,8 @@ export class ViewportBindingNaturalIosEmbed_ {
   /** @override */
   updatePaddingTop(paddingTop) {
     onDocumentReady(this.win.document, () => {
-      this.win.document.body.style.paddingTop = px(paddingTop);
+      this.win.document.body.style.borderTop =
```
Comment (cramforce) [inline src/service/viewport-impl.js]: This could also use a comment. And mention that we tried paddingTop

**Comment by cramforce** [comment]
LGTM

File: src/service/fixed-layer.js (hunk 1 above)
Comment (dvoytenko) [inline src/service/fixed-layer.js]: Done

File: src/service/viewport-impl.js (hunk 2 above)
Comment (dvoytenko) [inline src/service/viewport-impl.js]: Done

File: src/service/fixed-layer.js [hunk 3]
```diff
@@ -198,9 +200,18 @@ export class FixedLayer {
               element./*OK*/offsetWidth > 0 &&
               element./*OK*/offsetHeight > 0);
           // Transferability requires element to be fixed and top or bottom
-          // to be styled with `0`.
+          // to be styled with `0`. Also, do not transfer opaque or invisible
+          // elements - that's a lot of work for no benefit. Additionally,
+          // invisible/opaque elements used for "service" needs and thus best
```
Comment (jridgewell) [inline src/service/fixed-layer.js]: "opaque" is the wrong word here, I think you meant transparent?

**Pull request merged**